content/en/docs/how-tos/use-registries-in-build-farm.md

```diff
@@ -185,46 +172,9 @@ $ podman pull registry.build02.ci.openshift.org/ci-op-2c2tvgti/pipeline:src
 ## How do I access the latest published images for my component?
 
 If the `ci-operator` configuration for your component configures image [`promotion`](/docs/architecture/ci-operator/#publishing-container-images),
-output container images will be published to the central CI registry when changes are merged to your repository. Two main
+output container images will be published to QCI when changes are merged to your repository. Two main
 configurations are possible for promotion: configuring an `ImageStream` name and namespace or a namespace and a target tag.
 
-### Granting Pull Privileges
```

Comment:
We don't need this section anymore because of the new ServiceAccount token step, right?

Reply:
Exactly. Technically, the method in this section still works and we do not block the users from using it.
But we do not encourage them to use it as the created token is more desired as the users manages its lifecycle and rotation.